tiled/client/base.py

```diff
@@ -370,6 +380,99 @@ def update_metadata(self, metadata=None, specs=None):
 
         This is subject to change or removal without notice
 
+        Parameters
+        ----------
+        metadata : dict, optional
+            User metadata. May be nested. Must contain only basic types
+            (e.g. numbers, strings, lists, dicts) that are JSON-serializable.
+            Hint:
+                You can use `metadata_copy()` to retrieve a mutable copy of
+                the current metadata that can be passed here with modifications.
+            Note:
```

Comment:
The numpydoc standard supports a `Note` section. I'd suggest moving this out to keep the `Parameters` section streamlined.

I don't quite understand what this first sentence means. Maybe in the Note, explain, "This compares the input to the current metadata and constructs a JSON patch," and add additional example(s) to the Example section to illustrate `DELETE_KEY` and  this statement about relative paths.

Reply:
"relative paths" was a bad word choice on my part.  I will add examples to demonstrate what I actually mean.